Mu2eG4/geom/crv_counters_v08.txt

```diff
@@ -0,0 +1,408 @@
+//This geometry uses the same layout as v07 (same coordinates, etc.), but it splits the CRV-R1 sector into two sectors.
+//Therefore, this geometry is backward compatible with files that were simulated using v07 (except the coincidence data products).
+//This geometry requires the use of CRVResponse/fcl/prolog_v09.fcl.
+
+
+int   crs.nSectors = 23;
+int   crs.nLayers  = 4;
+
+vector<string> crs.sectorNames = {
+                       "R1",  //0   CRV-R modules first three modules at TS
+                       "R2",  //1   CRV-R modules upstream of cryo hole (except first three modules at TS)
+                       "R3",  //2   CRV-R module above cryo hole
+                       "R4",  //3   CRV-R module below cryo hole
+                       "R5",  //4   CRV-R modules downstream of cryo hole (except endcap)
+                       "R6",  //5   CRV-R modules at endcap
+                       "L1",  //6   CRV-L modules (except endcap)
+                       "L2",  //7   CRV-L modules at endcap
+                       "T1",  //8   CRV-TS modules (three upstream modules)
+                       "T2",  //9   CRV-TS modules (two downstream modules)
+                       "T3",  //10   CRV-T modules (four upstream modules)
+                       "T4",  //11  CRV-T modules (remaining 16 downstream modules)
+                       "E1",  //12  CRV-TS-Extension upstream module
+                       "E2",  //13  CRV-TS-Extension downstream module 
+                       "U",   //14  CRV-U modules
+                       "D1",  //15  CRV-D modules above beam pipe
+                       "D2",  //16  CRV-D module at -x of beam pipe
+                       "D3",  //17  CRV-D module at +x of beam beam pipe
+                       "D4",  //18  CRV-D module below beam pipe
+                       "C1",  //19  CRV-Cryo-Inner module upstream of cryo hole
```

Comment:
Do you want to disable C1 and C2?
Maybe in a prolog by setting LY to 0,  if you're not comfortable doing that here?

Reply:
I set the LY to 0 in the fcl file. Removing C1 and C2 entirely would make older art files unreadable.